pkg/kubelet/status/status_manager.go

```diff
@@ -675,9 +675,6 @@ func (m *manager) updateStatusInternal(pod *v1.Pod, status v1.PodStatus, forceUp
 	// Set PodReadyToStartContainersCondition.LastTransitionTime.
 	updateLastTransitionTime(&status, &oldStatus, v1.PodReadyToStartContainers)
 
-	// Set PodScheduledCondition.LastTransitionTime.
```

Comment:
so this will not be a problem for a version skew as the api version will be at least the same as kubelet version so will have the fix available. Is there any other weird cases of a version skew to worry about?